var/spack/repos/builtin/packages/py-tables/package.py

```diff
@@ -25,26 +26,35 @@ class PyTables(PythonPackage):
     variant("bzip2", default=False, description="Support for bzip2 compression")
     variant("lzo", default=False, description="Support for lzo compression")
 
-    # requirements.txt
-    depends_on("python@3.6:", when="@3.7.0:", type=("build", "run"))
-    depends_on("python@3.5:", when="@3.6.1:", type=("build", "run"))
-    depends_on("python@3.4:", when="@3.6.0:", type=("build", "run"))
-    depends_on("python@2.6:", type=("build", "run"))
+    # pyproject.toml
+    depends_on("py-setuptools@42:", when="@3.7:", type="build")
+    depends_on("py-setuptools", type="build")
+
+    # setup.py
+    depends_on("python@3.8:", when="@3.8:", type=("build", "run"))
 
-    depends_on("py-setuptools", type=("build", "run"))
-    depends_on("py-setuptools@42.0:", when="@3.7.0:", type=("build", "run"))
+    # requirements.txt
+    depends_on("py-cython@0.29.21:", when="@3.7:", type=("build", "run"))
     depends_on("py-cython@0.21:", type="build")
-    depends_on("py-cython@0.29.21:", when="@3.7.0:", type="build")
+    depends_on("py-numpy@1.19:", when="@3.8:", type=("build", "run"))
     depends_on("py-numpy@1.9.3:", type=("build", "run"))
     depends_on("py-numexpr@2.6.2:", type=("build", "run"))
-    depends_on("py-six@1.9.0:", when="@:3.5", type=("build", "run"))
-    depends_on("py-packaging", when="@3.7.0:", type=("build", "run"))
+    depends_on("py-blosc2@2.0", type=("build", "run"))
+    depends_on("py-packaging", when="@3.7:", type=("build", "run"))
+    depends_on("py-py-cpuinfo", when="@3.8:", type=("build", "run"))
+
     # tables/req_versions.py
-    depends_on("hdf5@1.8.4:1.8", when="@:3.3")
     depends_on("hdf5@1.8.4:", when="@3.4.0:")
+    depends_on("hdf5@1.8.4:1.8", when="@:3.3")
+
+    # Historical dependencies
+    depends_on("py-six@1.9:", when="@:3.5", type=("build", "run"))
+
+    # tables/req_versions.py
     # Versions prior to 3.3 must build with the internal blosc due to a lock
     # problem in a multithreaded environment.
-    depends_on("c-blosc@1.4.1:", when="@3.3.0:")
+    depends_on("c-blosc@1.11.1:", when="@3.8:")
+    depends_on("c-blosc@1.4.1:", when="@3.3:")
```

Comment:
```
depends_on("c-blosc2@2.5.0:", when="@3.8:")
```
and further down
```
        if "^c-blosc2" in self.spec:
            env.set("BLOSC2_DIR", self.spec["c-blosc2"].prefix)
```
see [req_versions.py#L14](https://github.com/PyTables/PyTables/blob/v3.8.0/tables/req_versions.py#L14)

Sorry overlooked this before, but I still get the 
```
  .. ERROR:: Unable to find the blosc2 library.
```